docs/libraries/libraries-with-cli.md

```diff
@@ -0,0 +1,660 @@
+# Writing Libraries with Cross Platform Tools
+
+By [Phillip Carter](https://github.com/cartermp)
+
+**Some details are subject to change as the toolchain evolves.**
+
+Targeting .NET Core for libraries can be done a number of ways, depending on the platform(s) you wish to support.  Sometimes Visual Studio is not available or a preferred development option.  .NET Core has a set of command line tools you can use instead of Visual Studio.
+
+* [Prerequisities](#prerequisites)
+* [How to target .NET Core](#how-to-target-net-core)
+* [How to target .NET Framework](#how-to-target-net-framework)
+* [How to target a Portable Class Library (PCL)](#how to target a portable class-library-pcl)
+* [How to cross-compile for .NET Core and .NET Framework](#how-to-cross-compile-for-net-core-and-net-framework)
+* [How to test libraries on .NET Core](#how-to-test-libraries-on-net-core)
+* [How to create a NuGet Package with your Library](#how-to-create-a-nuget-package-with-your-library)
+
+## Prerequisites
+
+You must have .NET Core installed on your machine.  This guide will focus on command line tooling because they work cross-platform and are what Visual Studio uses internally.  You have two options:
+
+* [DNX](http://aspnet.readthedocs.org/en/latest/getting-started/index.html), which has shipped with Visual Studio 2015 Update 1
+
+* [.NET CLI](http://dotnet.github.io/getting-started/), the tooling which will eventually replace DNX
+
+Either toolchain can be used to build and publish libraries today.
+
+The sections of this document dealing with older frameworks or Portable Class Libraries (PCLs) need the .NET Framework installed.  This is only supported on Windows.  
```

Comment:
Try to avoid use "older" unless that's really what you mean. It isn't here. In this case "older" also applies to 4.6.1, which is not very "old".

How about:

> The sections of this document dealing with the .NET Framework ...